packages/web/src/js/Toast.ts

```diff
@@ -29,48 +66,187 @@ class Toast extends BaseComponent {
   constructor(element: SpiritElement, config?: SpiritConfig) {
     super(element, config);
 
+    this.container = this.getContainer();
+    this.isShown = this.checkShownState();
     this.triggers = this.getTriggers();
+  }
 
-    this.isShown = this.checkShownState();
+  checkShownState(): boolean {
+    if (this.element) {
```

Comment:
Maybe the usage of a ternary operator with one return will be better in this case.